src/pdfflow/configflow.py

```diff
@@ -3,23 +3,24 @@
 """
 # Most of this can be moved to a yaml file without loss of generality
 import os
-
+import logging
 import numpy as np
+import tensorflow as tf
 
 # Set TF to only log errors
 os.environ.setdefault("TF_CPP_MIN_LOG_LEVEL", "1")
-import tensorflow as tf
 
 # uncomment this line for debugging to avoid compiling any tf.function
 # tf.config.run_functions_eagerly(True)
 
-def run_eager(flag = True):
-    """ Wraper around `run_functions_eagerly` """
-    tf.config.run_functions_eagerly(flag)
+def run_eager(flag=True):
+    """ Wrapper around `run_functions_eagerly` """
+    if '2.2' in tf.__version__:
+        tf.config.experimental_run_functions_eagerly(flag)
+    elif '2.3' in tf.__version__:
+        tf.config.run_functions_eagerly(flag)
```

Comment:
Probably you don't want to add a further dependency, however a really generic and simple way to parse a version is using [`semver`](https://pypi.org/project/semver/), since now a lot of projects are compliant with `semver`

Reply:
Hoping than google will be compliant with anyone other than themselves is a big hope. In any case, in this case we just want to avoid calling 2.2 but won't need to check beyond that.